core/src/main/scala/org/apache/spark/ui/ToolTips.scala

```diff
@@ -29,14 +29,15 @@ private[spark] object ToolTips {
   val SHUFFLE_READ_BLOCKED_TIME =
     "Time that the task spent blocked waiting for shuffle data to be read from remote machines."
 
-  val INPUT = "Bytes read from Hadoop or from Spark storage."
+  val INPUT = "Bytes and # records read from Hadoop or from Spark storage."
```

Comment:
Technically it's # bytes as well right?  I think this would be clearer as just "Bytes and records read".